pzagor2/TweenTime
# Update surface styles

Element rows:

- [x] Indentation
  - [x] Follow latest tree given from outside
- [x] Line height
- [x] Color samples followed by element names
- [x] Font family
- [x] Font size

Property rows (position, size, opacity):

- [x] Indentation
  - [x] Follow latest tree given from outside
- [x] Line height
- [x] Font family
- [x] Font size

Timeline:

- [x] Background color
- ~~[ ] Vertical lines should be white except for just on seconds~~ seems we don't have the feature to show lines between sec and sec
- [x] Horizontal lines should be white

Keys on timeline:

- [x] Show ribbons between key pairs which values are not same
  - [x] Update ribbon length when key positions are changed
- [x] bug: when you click a key, it jumps away

Others which should be done in another branches:

- bug: vertical lines don't follow scrolling
- Header
  - Add time unit "sec"
  - Switch UI for enable/disable timeline
- Loop
- Timeline/parallax (what are these?)
